package.json

```diff
@@ -6,7 +6,7 @@
   },
   "engines": {
     "npm": ">=6",
-    "node": "12"
+    "node": "15"
```

Comment:
Is this version of node necessary for eslint? I think it might conflict with gihub actions workflow where the used version is 12.8